conn/bind_std.go

```diff
@@ -268,8 +268,8 @@ func (s *StdNetBind) receiveIP(
 	}
 	for i := 0; i < numMsgs; i++ {
 		msg := &(*msgs)[i]
-		sizes[i] = msg.N
-		if sizes[i] == 0 {
+		sizes[i][0], sizes[i][1] = 0, msg.N
```

Comment:
minor suggestion, optional: move the continue branch below further up, and make the condition look at msg.N instead of the sizes array? As written here it took me a beat to translate the branch condition to "oh, skip empty messages".